src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -345,6 +346,12 @@ public GHPullRequestReviewComment createReviewComment(String body, String sha, S
                 .to(getApiRoute() + COMMENTS_ACTION, GHPullRequestReviewComment.class).wrapUp(this);
     }
 
+    public void requestReviewers(List<GHUser> reviewers) {
+        return new Requester(root).method("POST")
+                .with("reviewers", toLogins(reviewers))
```

Comment:
@twcurrie any plans to also support `team_reviewers`?

https://developer.github.com/v3/pulls/review_requests/#create-a-review-request

Reply:
Sure, is this a blocker? (Are you even able to merge this?)

Reply:
> (Are you even able to merge this?)

No, I'm simply advocating for `team_reviewers` &mdash; I was going to open my own PR for https://developer.github.com/v3/pulls/review_requests/#create-a-review-request but then I saw this one.

I think we need @kohsuke to review and merge.